RELEASING.md

```diff
@@ -23,7 +22,20 @@ git checkout -B 3.17
 git push --set-upstream origin 3.17
 ```
 
-## 3) Smoke test: ensure Docker Compose starts from scratch
+### Update the image tags
+
+> ⚠️ If you are using the Sourcegraph release tooling, this step will be done for you in the PR it creates.
+
+In the latest release branch you created:
+
+1. Run `tools/update-docker-tags.sh $VERSION`
+2. Confirm the diff shows the image tags being updated to the version you expect, and push directly to the release branch.
+
+### Cherry-pick image update to master
+
+Cherry-pick the commit where you update images on the release branch back into `master`.
```

Comment:
so the release tooling will make the PR to update images in only the release branch? is there a reason we can't invert this and have it do it against `master`, and then create the release branch from that commit in `master`?

Reply:
I just felt it aligned better with the `deploy-sourcegraph` scheme, and I've been trying to have the tool create "final" PRs - e.g. PRs where the merge _is_ the final state of the release (thus, the release campaign becomes a good snapshot of the state of a release).

IMO updating `master` is a secondary concern by comparison (since nobody should be using it), so if a delayed manual step must happen I'd rather it be the `master` update than the release branch update

Reply:
My concern here is that someone will forget to cherry-pick the commit. But it sounds like you have grander plans here and are aware of that, so I'm happy :)